src/pybind/mgr/rbd_support/schedule.py

```diff
@@ -387,17 +387,18 @@ def load(self,
 
         schedule_cfg = self.handler.module.get_module_option(
             self.handler.MODULE_OPTION_NAME, '')
+        schedule_local_cfg = self.handler.module.get_localized_module_option(
+             self.handler.MODULE_OPTION_NAME, '')
```

Comment:
nit: is indentation correct here? In github it looks like there is an extra space.

Reply:
Yup, here is flake8 complaint:
```
rbd_support/schedule.py:391:14: E126 continuation line over-indented for hanging indent
1     E126 continuation line over-indented for hanging indent
```